src/components/Table/Table.test.tsx

```diff
@@ -1,36 +1,31 @@
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 import React from "react";
 
 import Table from "./Table";
 
 describe("Table", () => {
   it("renders", () => {
-    const wrapper = shallow(
+    render(
       <Table>
         <thead></thead>
       </Table>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(screen.getByRole("grid")).toMatchSnapshot();
```

Comment:
I guess we could use data id or container to get root element.

Looking at this, I'm not sure if `grid` is the right role for a table. It's meant for interactive grids, not static tables.

Reply:
I've changed this to use the container.